feature-policy/resources/feature-policy-payment.html

```diff
@@ -0,0 +1,16 @@
+<script>
+'use strict';
+
+window.onload = function() {
+  var supportedInstruments = [ { supportedMethods: [ 'visa' ] } ];
```

Comment:
This fix will be in another CL: https://chromium-review.googlesource.com/c/533953